### ISSUE TITLE
AccessToken#video_grant is not publicly assignable

### ISSUE THREAD

**Issue by ianhirschfeld** [livekit/server-sdk-ruby#67]
The recommended method for adding a video grant to an access token is via assignment to the `video_grant` attribute, however that attribute is not accessible and throws `NoMethodError (undefined method video_grant=' for an instance of LiveKit::AccessToken)`. The only way to add a video grant is to use `add_grant` which is marked as deprecated.

The readme has this code snippet:

```ruby
require 'livekit'

token = LiveKit::AccessToken.new(api_key: 'yourkey', api_secret: 'yoursecret')
token.identity = 'participant-identity'
token.name = 'participant-name'
token.video_grant = LiveKit::VideoGrant.new(roomJoin: true, room: 'room-name')
token.attributes = { "mykey" => "myvalue" }
puts token.to_jwt
```

within the sdk:
```ruby
# Deprecated, use video_grant = instead
def add_grant(video_grant)
  if video_grant.is_a?(Hash)
    video_grant = VideoGrant.from_hash(video_grant)
  end
  self.video_grant = video_grant
end
```

a working code snippet:
```ruby
token = LiveKit::AccessToken.new(api_key: 'yourkey', api_secret: 'yoursecret')
token.identity = 'participant-identity'
token.name = 'participant-name'
token.add_grant(roomJoin: true, room: 'room-name')
token.attributes = { "mykey" => "myvalue" }
puts token.to_jwt
```

**Comment by davidzhao** [comment]
sorry about that, this is due to the latest version being unreleased.. cutting a build for it now

**Comment by davidzhao** [comment]
build released, please update to 0.8.0 and give it another try

**Comment by ianhirschfeld** [comment]
> build released, please update to 0.8.0 and give it another try

Thanks so much for the quick response! Updating to v0.8.0 _did_ fix it 🙌
